Jenkinsfile

```diff
@@ -40,6 +40,8 @@ pipeline {
           steps {
             buildImage('hugo', '0.81.0', 'apps/hugo/Dockerfile', ['HUGO_VERSION': '0.81.0'])
             buildImage('hugo_extended', '0.81.0', 'apps/hugo_extended/Dockerfile', ['HUGO_VERSION': '0.81.0'])
+            buildImage('hugo', '0.110.0', 'apps/hugo/Dockerfile', ['HUGO_VERSION': '0.110.0'])
+            buildImage('hugo_extended', '0.110.0', 'apps/hugo_extended/Dockerfile', ['HUGO_VERSION': '0.110.0'])
```

Comment:
Please replace the versions instead of duplicating the images. We don't need to build images for old versions, if nobody uses them.

Reply:
Done!